doc/cephadm/install.rst

```diff
@@ -6,6 +6,12 @@ Cephadm creates a new Ceph cluster by "bootstrapping" on a single
 host, expanding the cluster to encompass any additional hosts, and
 then deploying the needed services.
 
+Each host should be prepared as follows for deployment:
+
+- Allow SSH for the root user
+- For the first "bootstrapping" host and any other hosts intended to be monitors, set up passwordless SSH to all other hosts
+- Ensure that each host has a unique hostname (unique IP addresses are not sufficient for the instructions below)
```

Comment:
Did you run into this thing? Imaging having hosts with the same name is going to be super irritating for managing ceph.

Reply:
Using a vmware hypervisor, if you clone a VM it will have the same hostname.  It is just an alias for the IP anyway so it is easy to administer using IP addresses.

Reply:
uh, but then you should't be able to add the two host names. Feels a bit odd to express this in the docs, which I think is obvious. @zdover23  any thoughts?

Reply:
Right, that was the problem, adding a new host with the same hostname clobbers the old one.  It was obvious that this was going on by inspection, not so obvious why different IP addresses were not sufficient.

Reply:
> uh, but then you should't be able to add the two host names. Feels a bit odd to express this in the docs, which I think is obvious. @zdover23 any thoughts?

This VM-cloning-based method of deploying a cluster is not the method that I used to create the procedure that we have in these docs. That doesn't mean that @SLongofono is wrong and it doesn't mean that the procedure as written is wrong. It just means that these are two different approaches to deploying a cluster.

My intuition is that VM-cloning gotchas are the proper provenance of hypervisor-related docs, but I also think that this note is absolutely not objectionable and my vote (if we're suddenly in some kind of democracy) is that this gets ingested into the documentation, because it represents and communicates real experience gathered in the field.

This is a pretty good example of how habits and reflexes can differ and still be right.

Reply:
Also, I don't know what's up with github, but I'm not getting notifications that I've been @zdover23'd. So I'm using this comment to see if I can ping myself.

zdover23: @zdover23